specification/cognitiveservices/OpenAI.Inference/tspconfig.yaml

```diff
@@ -27,7 +27,7 @@ options:
     enable-sync-stack: true
     generate-tests: false
     custom-types-subpackage: "implementation.models"
-    custom-types: "FunctionCallModelBase,FunctionCallPreset,FunctionCallPresetFunctionCallModel,FunctionNameFunctionCallModel"
+    custom-types: "FunctionCallPreset"
```

Comment:
What's the effect of this change?

Reply:
It moves this model to impl. Maybe we can use `@internal`